src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -81,6 +81,22 @@
   private static final String CONNECTION_PASSWORD_DOC = "JDBC connection password.";
   private static final String CONNECTION_PASSWORD_DISPLAY = "JDBC Password";
 
+  public static final String CONNECTION_ATTEMPTS =
+          JdbcSourceConnectorConfig.CONNECTION_ATTEMPTS_CONFIG;
+  private static final String CONNECTION_ATTEMPTS_DOC
+          = "Maximum number of attempts to retrieve a valid JDBC connection. "
+          + "Must be a positive integer.";
+  private static final String CONNECTION_ATTEMPTS_DISPLAY = "JDBC connection attempts";
+  public static final int CONNECTION_ATTEMPTS_DEFAULT = 3;
```

Comment:
Maybe reuse the constants from `JdbcSourceConnectorConfig` for all of these, similar to the `QUOTE_SQL_IDENTIFIERS_CONFIG` family of constants?

Reply:
Done!